cmake/Options.cmake

```diff
@@ -10,7 +10,9 @@ include(BlasLapackOption)
 option(ENABLE_PACKAGING "Enable Packaging" OFF)
 option(ENABLE_PYTHON "Enable Python Wrapping" ON)
 option(USE_OMP "Use OpenMP" OFF)
+option(USE_GIFTI "Use GIFTI IO support" OFF)
 option(USE_VTK "Use VTK" OFF)
+option(USE_CGAL "Use CGAL meshing tools" OFF)
 option(BUILD_TESTING "Build the testing tree" ON)
 option(BUILD_DOCUMENTATION "Build the documentation" ON)
 
```

Comment:
Duplicated options with the UseGifti and UseVtk.cmake. I'd prefer that option are in their own files and that this file only includes the various option files. And the same for the superproject.